tg_bot/modules/sql/locks_sql.py

```diff
@@ -216,6 +232,60 @@ def get_restr(chat_id):
         SESSION.close()
 
 
+def get_whitelist(chat_id):
+    return CHAT_WHITELIST.get(str(chat_id), {})
+
+
+def add_whitelist(chat_id, url):
+    url = URL_REGEXP.search(url).group(3).lower()
```

Comment:
Multiple chaining will blow up in case of `None` return from the search, or the group.